emr_launcher.py

```diff
@@ -212,6 +205,58 @@ def handler(event: dict = {}, context: object = None) -> dict:
     return resp
 
 
+def add_command_line_params(cluster_config, correlation_id, s3_prefix):
+    """
+    Adding command line arguments to ADG and PDM EMR steps scripts. First if block in Try is for PDM and the second one
+    is for ADG.
+    """
+    try:
+        if (
+            next(
```

Comment:
Rather than deciding on the name of the steps the clusters have , can we have  static argument that doesn't  with values adg and pdm to know what is pdm and adg?